libnmstate/nm/team.py

```diff
@@ -96,11 +96,9 @@ def get_info(device):
 
 def _convert_teamd_config_to_nmstate_config(team_config):
     team_config.pop(TEAMD_JSON_DEVICE, None)
-    port_config = team_config.get(Team.PORT_SUBTREE)
-
-    if port_config:
-        team_port = _teamd_port_to_nmstate_port(port_config)
-        team_config[Team.PORT_SUBTREE] = team_port
+    port_config = team_config.get(Team.PORT_SUBTREE, {})
```

Comment:
Is `Team.PORT_SUBTREE` correct here? It seems to be the `ports` constant from the teamd config and not the Nmstate port subtree state here. Or is it?

Reply:
You are right, it is teamd state, not nmstate.

Reply:
I changed it to express this.
We return teamd state adjusted a bit, I hope I expressed it in the code so it will be clear.